### PR TITLE
antiword: test added

### PR DESCRIPTION
**Pull request by bfontaine**

### CODE REVIEW

File: Library/Formula/antiword.rb [hunk 1]
```diff
@@ -1,21 +1,24 @@
...
-
 class Antiword < Formula
-  homepage 'http://www.winfield.demon.nl/'
-  url 'http://www.winfield.demon.nl/linux/antiword-0.37.tar.gz'
-  sha1 '4364f7f99cb2d37f7d1d5bc14a335ccc0c67292e'
+  homepage "http://www.winfield.demon.nl/"
+  url "http://www.winfield.demon.nl/linux/antiword-0.37.tar.gz"
+  sha256 "8e2c000fcbc6d641b0e6ff95e13c846da3ff31097801e86702124a206888f5ac"
+
+  resource "sample.doc" do
```
Comment (DomT4) [inline Library/Formula/antiword.rb]: Is there any way to inline this with a `(testpath/"example.doc").write` or?

Comment (bfontaine) [inline Library/Formula/antiword.rb]: @DomT4 No, because it’s binary data.

**Comment by xu-cheng** [comment]
Thanks @bfontaine

File: Library/Formula/antiword.rb (hunk 1 above)
Comment (DomT4) [inline Library/Formula/antiword.rb]: Fair enough :+1:

**Pull request closed without merging**